pan3d/utils/common.py

```diff
@@ -357,7 +352,7 @@ def __init__(
 
         with self:
             with v3.VToolbar(
-                classes="pl-2",
+                classes="pl-2 d-flex",
```

Comment:
do you still need that when you remove the css load above (39-40)?

Reply:
No, it was a side effect of applying a patch.